### ISSUE TITLE
Feature request : Vite 5 / Rollup 4 support (fixes `unmet peer rollup@^1.20.0||^2.0.0||^3.0.0: found 4.9.*`)

### ISSUE THREAD

**Issue by MarArMar** [threedayAAAAA/vite-plugin-circular-dependency#5]
I am using Sveltekit v2, which requires Vite 5 and it seems to install also @rollup/pluginutils 5.0.2

Which makes these warnings at install time :
```
└─┬ vite-plugin-circular-dependency 0.2.1
  └─┬ @rollup/pluginutils 5.0.2
    └── ✕ unmet peer rollup@^1.20.0||^2.0.0||^3.0.0: found 4.9.6
```

Will report if I see it breaking

Thank you so much btw, it has been an extremely useful lib for me as it helps uncover deep problems involving circular deps ! 💛

**Comment by threedayAAAAA** [comment]
Thank you very much for reporting this issue, I think it is caused by a dependency on @rollup/pluginutils. This problem has been solved in version 5.0.5. I will update this dependency and release a new version. You can update to the latest version. Thanks again for raising this question

https://www.npmjs.com/package/@rollup/pluginutils/v/5.0.5?activeTab=code

**Comment by threedayAAAAA** [comment]
You can update the version to 0.3.1
